content/v2.0/influxdb-templates/create.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Create an InfluxDB template
+description: >
+  Use the InfluxDB UI and the `influx pkg export` command to create InfluxDB templates.
+menu:
+  v2_0:
+    parent: InfluxDB templates
+    name: Create a template
+    identifier: Create an InfluxDB template
+weight: 101
+v2.0/tags: [templates]
+---
+
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create InfluxDB templates.
+Add resources (buckets, Telegraf configurations, tasks, and more) in the InfluxDB
+UI and export then the resources as a template.
+
+{{% note %}}
+Templatable resources are scoped to a single organization, so the simplest way to create a
```

Comment:
A template's resources are associated w a single organization?
curious...are there templatable resources that **cannot** be scoped to a single organization?

Reply:
Everything in 2.0 is scoped to an organization except the "root" token and user, but neither of these are templatable.